projects/tensorflow/build.sh

```diff
@@ -116,6 +111,8 @@ then
   declare -r REMAP_PATH=${OUT}/proc/self/cwd
   mkdir -p ${REMAP_PATH}
   rsync -ak ${SRC}/tensorflow ${REMAP_PATH}
+  rsync -ak ${SRC}/third_party ${REMAP_PATH}
+  mkdir -p ${OUT}/coverage/report/coverage
```

Comment:
I don't think this line is needed. It's going to be created at some step if the steps before this have succeeded. Could you please remove it?

Reply:
Should be removed now